dashboard/web/index.html

```diff
@@ -24,17 +24,30 @@
   <link rel="icon" type="image/png" href="favicon.png"/>
 </head>
 <body>
-  <script src="main.dart.js" type="application/javascript"></script>
-  <script>
-    window.flutterConfiguration = {
-      canvasKitBaseUrl: "/canvaskit/"
-    };
-    (function(w,d,s,g,js,fjs){
-      g=w.gapi||(w.gapi={});g.analytics={q:[],ready:function(cb){this.q.push(cb)}};
-      js=d.createElement(s);fjs=d.getElementsByTagName(s)[0];
-      js.src='https://apis.google.com/js/platform.js';
-      fjs.parentNode.insertBefore(js,fjs);js.onload=function(){g.load('analytics')};
-    }(window,document,'script'));
-    </script>
+<script src="main.dart.js" type="application/javascript"></script>
+<script>
+  window.flutterConfiguration = {
+    canvasKitBaseUrl: "/canvaskit/"
+  };
+  (function(w,d,s,g,js,fjs){
+    g=w.gapi||(w.gapi={});g.analytics={q:[],ready:function(cb){this.q.push(cb)}};
+    js=d.createElement(s);fjs=d.getElementsByTagName(s)[0];
+    js.src='https://apis.google.com/js/platform.js';
+    fjs.parentNode.insertBefore(js,fjs);js.onload=function(){g.load('analytics')};
+  }(window,document,'script'));
+
+  function checkTreeStatus(){
```

Comment:
Can we use `dart:html` from Flutter for Web? It feels wrong doing this from JS. @CaseyHillers do you know?